requirements/docs.txt

```diff
@@ -12,3 +12,7 @@ ipykernel>=5.1.0, <6.0.0 # required for executing notebooks via nbsphinx
 ipython>=7.2.0, <8.0.0 # required for executing notebooks nbsphinx
 # pandoc
 # pandoc==1.16.02 # NB: as this is not a Python library, it should be installed manually on the system or via a package manager such as `conda`
+# alibi_detect dependencies (these are installed on ReadTheDocs so are not mocked)
+numpy>=1.16.2, <2.0.0
+pandas>=0.23.3, <2.0.0
+matplotlib>=3.0.0, <4.0.0
```

Comment:
Can you remind me the reason we want these on RTD? To render types better or another reason?

Reply:
Realized that this is explained in the `conf.py` - RTD 3.9 includes these deps.